frontend/src/reviews/ReviewComments.tsx

```diff
@@ -80,18 +83,29 @@ export default function ReviewComments({ release, accessRequest, isEdit }: Revie
     decisionsAndComments.sort(sortByCreatedAtAscending)
     return decisionsAndComments.map((response) => {
       if (response.kind === ResponseKind.Review) {
-        return <ReviewDecisionDisplay key={response.createdAt} response={response} modelId={modelId} />
+        return (
+          <ReviewDecisionDisplay
+            key={response.createdAt}
```

Comment:
If you do update `ResponseInterface` to have `_id`, we can use that `_id` as a guaranteed unique key.